### DISCUSSION TITLE
Chess issue? Mixed version snafu?

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#117] [General]
Install-product -s -f games <-- used to install

The game seems to work fine on the console, but which version?  Haven't tried explain in the console though.  Explain,all did work when just 4.9.

![vYwdipaKah_094327](https://github.com/kej715/DtCyber/assets/136194499/9eb4703e-d0e6-4af6-950e-0efbff2e32e0)

**Comment by kej715** [comment]
Apparently, that’s what the Chess 4.9 code actually has in it. Perhaps the original authors forgot to update the interactive prompt. When run from the system console, which is probably how it was run when competing in serious tournaments. The version number displayed matches the source version.On Jan 8, 2024, at 12:45 PM, KC5VDJ ***@***.***> wrote:﻿
Install-product -s -f games <-- used to install.
vYwdipaKah_094327.png (view on web)

**Comment by KC5VDJ** [comment]
Makes sense. Noticing that p-6000 4.1 may have similar internal issues (didn't bump all instances, including in comments, he wasn't focusing on the comments).

**Comment by KC5VDJ** [comment]
"Explain,all" did work in terminal session previous to the other versions being added.  Even with the 4.5 bit explained, there is still this.

**Comment by kej715** [comment]
Chess 4.9 has not been changed. The other versions were added without
changing 4.9 in any way. Note that if you specify an alternate output file,
EXPLAIN,ALL will issue full help text to the alternate output file. For
example, start the program as in:

    /chess,input,out,library

then when Chess prompts for input, enter EXPLAIN,ALL followed by the END
command, as in:

    ? EXPLAIN,ALL
    ? END

The file named OUT will contain the full help listing.
